src/Orchard/Services/IHtmlFilterRunner.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using Orchard.ContentManagement;
+
+namespace Orchard.Services {
+    public interface IHtmlFilterRunner : IDependency {
```

Comment:
I don't like the name `Runner`. We don't have such things right now. We have `Processor`, `Coordinator`, probably other choices but I don't like `runner`.